content/en/docs/components/katib/katib-config.md

```diff
@@ -85,6 +85,21 @@ a default value is set automatically.
    - `cpu = 500m`
    - `ephemeral-storage = 5Gi`
 
+   It is possible to not request the `ephemeral-storage` resource for the
+   metrics collector's container. For example to use
+   [GKE nodepool autoscalers](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
+   To do that, set the negative values for the `ephemeral-storage` requests and
+   limits in your Katib config:
```

Comment:
The second sentence is not complete, I think.
The third sentence is not clear about what "that" is.
Can you revise them please?

Reply:
Sure, @8bitmp3.
What do you think about this:
```
 You can run your metrics collector container without requesting
 the `ephemeral-storage` resource from the Kubernetes cluster.
 For instance, if you want to use
 [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
 for your Katib experiments. To remove the `ephemeral-storage` resource from
 the metrics collector container, set the negative values for the
 `ephemeral-storage` requests and limits in your Katib config:
```